cypress/support/loginCommands.ts

```diff
@@ -0,0 +1,16 @@
+export const loginWithUsername = () => {
+  cy.visit("https://smart-data-entry-ui.vercel.app/auth")
+
+  cy.get("#supertokens-root") // replace with the ID of your shadow DOM host element
+    .shadow()
+    .find('input[name="email"]')
+    .type("rugvedij1804@gmail.com")
+
+  cy.get("#supertokens-root")
+    .shadow()
+    .find('input[name="password"]')
+    .type("Rugvedi18")
```

Comment:
Use env vars for sensituve values such as passwords

Reply:
done